templates/theme/styles/base.less

```diff
@@ -277,6 +279,7 @@ atom-text-editor .search-results .marker.current-result .region,
     font-style: italic;
   }
 
+  &.heading,
   &.heading .punctuation.definition.heading {
```

Comment:
I have no idea. Just remove it, and hope nobody notices anything?
From the default languages, the only one with `.heading` is markdown, right?